specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/devices.json

```diff
@@ -0,0 +1,743 @@
+            "Hmi",
+            "Generic"
+          ],
+          "x-ms-enum": {
+            "name": "SlotType",
+            "modelAsString": true
+          },
+          "description": "Slot type."
+        },
+        "hardwareVendor": {
+          "type": "string",
+          "description": "Hardware vendor of the slot."
+        },
+        "hardwareRevision": {
+          "type": "string",
+          "description": "Hardware revision of the slot."
+        },
+        "firmwareVersion": {
+          "type": "string",
+          "description": "Firmware revision of the slot."
+        },
+        "model": {
+          "type": "string",
+          "description": "Model of the slot."
+        },
+        "moduleVersion": {
+          "type": "string",
+          "description": "Module version of the slot."
+        },
+        "serialNumber": {
+          "type": "string",
+          "description": "Serial number of the slot."
+        },
+        "productCode": {
+          "type": "string",
+          "description": "Product code of the slot."
+        },
+        "cpes": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of Common Platform Enumeration (CPE) of the slot",
+          "items": {
+            "type": "string"
+          }
+        },
+        "additionalData": {
+          "type": "object",
+          "description": " A bag of fields which extends the slot information."
+        }
+      }
+    },
+    "Nic": {
+      "type": "object",
+      "description": "Network interface card properties",
+      "properties": {
+        "lastSeen": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Last seen time"
+        },
+        "ipv4Address": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Ipv4 address"
+        },
+        "ipv6Address": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Ipv6 address"
+        },
+        "macAddress": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Mac address"
+        },
+        "networkAdapterVendor": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Network adapter vendor."
+        },
+        "macCertainty": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Guess",
+            "Certain"
+          ],
+          "x-ms-enum": {
+            "name": "MacCertainty",
+            "modelAsString": true
+          },
+          "description": "Indicates whether the association of the mac to the ip address is certain or a guess."
+        },
+        "vlans": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of virtual LAN (vlan)",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Sensor": {
```

Comment:
Consider renaming this type.
According to the name, it could be confused with the actual sensor resource, but the data exposed here isn't really the full sensor resource

Reply:
I choose to leave this name